packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -51,31 +92,76 @@ export default defineComponent({
 	data() {
 		return {
 			editor: {} as EditorView,
+			expressionsDocsUrl: EXPRESSIONS_DOCS_URL,
+			isFocused: false,
+			skipSegments: ['Statement', 'CompositeIdentifier', 'Parens'],
```

Comment:
This is weird, I added this because some parts of complex queries that I tested (using nested and aliased selects) were duplicated in the preview dropdown. Now tried removing it and all seems good. Probably was caused by something else then 🤔

Reply:
Turns out these are `CompositeIndentifier`s:
![SCR-20230619-kks](https://github.com/n8n-io/n8n/assets/2598782/8e0c1a2f-73d4-40dd-9d8b-42ad2281679e)